### PR TITLE
Add issueCredential route.

### PR DESCRIPTION
**Pull request by mattcollier**
First cut of issuer.  Where do we go from here?

It appears that all the other endpoints are optional.  Which of those should be implemented here?

https://w3c-ccg.github.io/vc-issuer-http-api/index.html#/

### CODE REVIEW

File: test/mocha/mock-credential.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      "givenName": "schema:givenName",
+      "familyName": "schema:familyName",
+      "parent": {"@id": "ex:parent", "@type": "@id"},
+      "referenceId": "ex:referenceId",
+      "documentPresence": "ex:documentPresence",
+      "evidenceDocument": "ex:evidenceDocument",
+      "spouse": "schema:spouse",
+      "subjectPresence": "ex:subjectPresence",
+      "verifier": {"@id": "ex:verifier", "@type": "@id"}
+    }
```
Comment (msporny) [inline test/mocha/mock-credential.json]: Almost all of this is unnecessary for the VC being produced by the test... can we remove the terms that are not used?

Comment (mattcollier) [reply test/mocha/mock-credential.json]: There's a FIXME related to this.  What are the requirements about context support?  Currently the `https://www.w3.org/2018/credentials/examples/v1` is not part of our bedrock document loader framework.

File: lib/http-credentials.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  const {routes} = cfg;
+
+  app.post(
+    routes.issueCredential,
+    // TODO: add validation
+    asyncHandler(async (req, res) => {
+      const {
+        credential,
+        // FIXME: what are options used for?
+        // options,
```
Comment (msporny) [inline lib/http-credentials.js]: See: https://w3c-ccg.github.io/vc-issuer-http-api/index.html#/internal/issueCredential

Options are arguments passed to the issuance process that are used to modify the signed output. For example, `assertionMethod` is an option that specifies which key the requestor would like to be used.

File: test/mocha/20-interop-issuer.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+const api = create({
+  baseURL: `${config.server.baseUri}/credentials`,
+  httpsAgent,
+  timeout: 1000,
+});
+
+// FIXME: using embedded context:
+// https://www.w3.org/2018/credentials/examples/v1
+// is this context supposed to be supported in a documentLoader
```
Comment (msporny) [inline test/mocha/20-interop-issuer.js]: No, we don't need to include it in the documentLoader... just specify the extra terms via an embedded context in the tests.

**Comment by mattcollier** [comment]
Replaced with https://github.com/digitalbazaar/bedrock-vc-issuer/pull/30 closing.

**Pull request closed without merging**